src/Console/Command/SetupCICommand.php

```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return self::FAILURE;
         }
 
-        if ($ci === CiDetector::CI_GITHUB_ACTIONS) {
-            $rectorWorkflowFilePath = getcwd() . '/.github/workflows/rector.yaml';
-            if (file_exists($rectorWorkflowFilePath)) {
-                $this->symfonyStyle->warning('The "rector.yaml" workflow already exists');
+        if ($ci !== CiDetector::CI_GITHUB_ACTIONS) {
+            $noteMessage = sprintf(
+                'Only Github Action is supported for now.%sCreate an issue to add your CI %s',
+                PHP_EOL,
+                'https://github.com/rectorphp/rector/issues/'
+            );
+
+            $this->symfonyStyle->note($noteMessage);
+            return self::SUCCESS;
+        }
+
+        $rectorWorkflowFilePath = getcwd() . '/.github/workflows/rector.yaml';
+        if (file_exists($rectorWorkflowFilePath)) {
+            $response = $this->symfonyStyle->ask('The "rector.yaml" workflow already exists. Overwrite it?', 'Yes');
+            if (! in_array($response, ['y', 'yes', 'Yes'], true)) {
+                $this->symfonyStyle->note('Nothing changed');
                 return self::SUCCESS;
             }
+        }
 
-            $currentRepository = $this->resolveCurrentRepositoryName(getcwd());
-            if ($currentRepository === null) {
-                $this->symfonyStyle->error('Current repository name could not be resolved');
+        $currentRepository = $this->resolveCurrentRepositoryName(getcwd());
+        if ($currentRepository === null) {
+            $this->symfonyStyle->error('Current repository name could not be resolved');
 
-                return self::FAILURE;
-            }
+            return self::FAILURE;
+        }
 
-            $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
+        $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
```

Comment:
**🎯 suggestion:** I usually prefer this notation for readability

```suggestion
        $workflowTemplate = FileSystem::read(dirname(__DIR__, 3) . '/templates/rector-github-action-check.yaml');
```

Reply:
This actually shuts the PHPStan check for missing file.
Also is dynamic over static value.